_posts/newsletters/2022-01-31-newsletter.md

```diff
@@ -0,0 +1,205 @@
+25. [Digital Library Software Engineer](https://sjobs.brassring.com/TGnewUI/Search/home/HomeWithPreLoad?partnerid=25620&siteid=5494&PageType=JobDetails&jobid=4416687): Arizona State University, Tempe, AZ *Posted: Dec 08, 2021*
+26. [Research Software Engineer (Entry Level)](https://rand.wd5.myworkdayjobs.com/en-US/External_Career_Site/job/Santa-Monica-CA-Greater-Los-Angeles-Area/Research-Software-Engineer--Entry-Level-_R772): RAND: Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington D.C. *Posted: Dec 07, 2021*
+27. [Applications Developer, User Interface](https://phf.tbe.taleo.net/phf03/ats/careers/v2/viewRequisition?org=CALTECH&cws=37&rid=6231): Caltech, Pasadena, CA *Posted: Nov 16, 2021*
+28. [Applications Developer, Data Systems](https://phf.tbe.taleo.net/phf03/ats/careers/v2/viewRequisition?org=CALTECH&cws=37&rid=6230): Caltech, Pasadena, CA *Posted: Nov 16, 2021*
+29. [Assistant Director, Penn State's Applications of Artificial Intelligence and Machine Learning to Industry (AIMI) Center - Institute for Computational and Data Sciences (ICDS)](https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/Assistant-Director--Penn-State-s-Applications-of-Artificial-Intelligence-and-Machine-Learning-to-Industry--AIMI--Center_REQ_0000021090-1): State College, Pennsylvania *Posted: Nov 10, 2021*
+30. [ICDS Technical Director - Institute for Computational and Data Sciences (ICDS)](https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/ICDS-Technical-Director_REQ_0000020689): State College, Pennsylvania *Posted: Nov 09, 2021*
+31. [System Engineering and Integration Lead - Institute for Computational and Data Sciences (ICDS)](https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/System-Engineering-and-Integration-Lead_REQ_0000020693): State College, Pennsylvania *Posted: Nov 09, 2021*
+32. [Operations Team Lead - Institute for Computational and Data Sciences (ICDS)](https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/Operations-Lead_REQ_0000020695-1): State College, Pennsylvania *Posted: Nov 09, 2021*
+33. [Researcher](https://jobs.hr.wisc.edu/en-us/job/511080/researcher): University of Wisconsin-Madison, Madison, WI *Posted: Oct 22, 2021*
+
+<a name="news"></a>
+# **Community News**
+
+* The UNESCO [Recommendation on Open Science](https://click.mailerlite.com/link/c/YT0xODM3MjgxNzE3NjYwOTQ5MDQ2JmM9bThhNCZiPTgyODY5NTY2OSZkPXczYTh2MXA=._oy_Gb9yAAS4Ino8P6O-u0yNpChfhrZlVzALpCn3SAU) has been adopted by its member states. This recommendation defines open scientific knowledge as including software and source code. In particular, the recommendation argues for users to gain free access to open source software and source code in a timely and user-friendly manner, in human- and machine-readable and modiﬁable format, under an open license.
```

Comment:
I think the site says UNESDOC and not UNESCO?
```suggestion
* The UNESDOC [Recommendation on Open Science](https://click.mailerlite.com/link/c/YT0xODM3MjgxNzE3NjYwOTQ5MDQ2JmM9bThhNCZiPTgyODY5NTY2OSZkPXczYTh2MXA=._oy_Gb9yAAS4Ino8P6O-u0yNpChfhrZlVzALpCn3SAU) has been adopted by its member states. This recommendation defines open scientific knowledge as including software and source code. In particular, the recommendation argues for users to gain free access to open source software and source code in a timely and user-friendly manner, in human- and machine-readable and modiﬁable format, under an open license.
```

Reply:
It's the United  Nations  Educational,  Scientific  and  Cultural Organization (UNESCO)